### DISCUSSION TITLE
Error: no such subcommand 'tauri-bundler'

### DISCUSSION THREAD

**Discussion by YairLevi** [tauri-apps/tauri#4890] [Q&A] [answered]
Hi, tried to build my project and this popped up:
![image](https://user-images.githubusercontent.com/79797419/183400469-ba5523f0-dc7d-47fb-a9b0-1f78618b5534.png)

Tried to install tauri-bundler but tells me: 
![image](https://user-images.githubusercontent.com/79797419/183400651-046c97e0-91a8-4b74-ac87-073d77a54d8e.png)

So what do I from here?

node 16.0.0
npm 7.10.0
cargo 1.62.1
OS windows 10

**Comment by FabianLars** [comment, marked as answer]
Did you see my answer at the [other repo](https://github.com/tauri-apps/realworld/issues/3)?

Please start your project from scratch again following our docs on our website. For example, tauri-bundler is not even a thing anymore (in this form).

**Reply by YairLevi** [reply]
My bad, didn't see the edit. I will try, thanks!
